it/BaseISpec.scala

```diff
@@ -0,0 +1,37 @@
+import org.jsoup.Jsoup
```

Comment:
Should all of these classes be inside a package?

I thought that classes defined with no package cannot be imported into another class, e.g.
```
packge random

class  RandomISpec extends BaseISpec {
...
}
```
would not compile because `BaseISpec` could not be found.

Reply:
I can add a package.

But are we expecting any packages?

I had imagined only having one `ISpec` for each screen, in that folder.

Reply:
OK, I don't mind keeping it all at top level for now. Maybe in the future we'll have separate sets of integration tests, and we can split it out then. But no problem for now.